selfdrive/ui/qt/offroad/networking.cc

```diff
@@ -148,17 +150,22 @@ AdvancedNetworking::AdvancedNetworking(QWidget* parent, WifiManager* wifi): QWid
   list->addItem(new SshToggle());
   list->addItem(new SshControl());
 
+  int prime_type = uiState()->prime_type;
+  bool gsmVisible = prime_type == PrimeType::NONE || prime_type == PrimeType::LITE;
```

Comment:
don't need to do this in the constructor, and you don't want to duplicate the logic anyway. just hide them by default. primeTypeChanged should be emitted shortly after this anyway